Automattic/wp-calypso
# Jetpack Cloud: Add Jetpack VideoPress Product

#### Changes proposed in this Pull Request

* Adds VideoPress as a new product to the pricing grid
* Moves CRM card to the bottom area alongside `Jetpack Free`
* Adds a thank you dialog to be displayed after purchase of Jetpack VideoPress product.

Screenshots:
![jetpack cloud localhost_3000_pricing (1)](https://user-images.githubusercontent.com/789137/132528052-c4ad3437-00d3-4fb7-802c-4bd8e841abe3.png)

<img width="1033" alt="Screen Shot 2021-09-15 at 12 22 10 PM" src="https://user-images.githubusercontent.com/789137/133489597-e55d814e-071e-41c7-a5f0-f447d9bda740.png">

<img width="1273" alt="Screen Shot 2021-09-15 at 12 07 29 PM" src="https://user-images.githubusercontent.com/789137/133489621-cb721ebe-8c01-4ab7-96a8-547a2987cb3b.png">

#### Testing instructions

* `CALYPSO_ENV=jetpack-cloud-development yarn start`
* View `/pricing`, and you should see VideoPress as an option.
* Clicking the `Get VideoPress` link should take you to the VideoPress checkout.
* After completing the purchase, kill the server and run `yarn start` to start regular calypso.
* Simulate the `Thank You` experience by loading: `http://calypso.localhost:3000/plans/my-plan/{your-jp-site-slug}?product=jetpack_videopress&thank-you`
* Visit `http://calypso.localhost:3000/plans/my-plan/{your-jp-site-slug}` and you should see Jetpack VideoPress in the `My solutions` section.

Related to pxWta-Zz-p2
